docs/dev/releases.md

```diff
@@ -0,0 +1,169 @@
+        
+        git checkout develop
+        git checkout -b release/<version number>
+
+3. See Pre-Release Checklist to make some minor changes for the pull request
+4. Now create a pull request for the release branch, should merge into `master`
+5. Add the following to the PR info: 
+    - **Title**: Release pyDARN <version>
+    - **Scope**: add information on the main focus of this release
+    - **Include link to readthedocs** on this release (ask for pyDARN lead developer or DAWG Chair to do this)
+    - **Changes:** history of what has changed, look at the [milestones closed tasks](https://github.com/SuperDARN/pydarn/milestones?state=open)
+    - **Approvals:** 3 (reviews+testing)
+    - **Testing:** Using Github Checklists to include what needs to be tested, tested on what kind of data, and operating systems
+        - For example: 
+            - Test summary plotting 
+            - Test fan plots 
+            - test grid plotting 
+            - test grabbing hardware information 
+            - test on North and South hemisphere data
+            - test on Borealis data 
+            - test using RPM linux machine (OpenSuse, Fedora, RedHat)
+            - test using Debian linux machine (Ubuntu)
+            - test using MacOsX
+            - test using Windows 
+
+
+
+### Pre-Release Checklist 
+
+!!! IMPORTANT
+    Do this before the pull request to make it part of the review process 
+
+- Update `.zenod.json` file is updated to reflect the [DAWG authorship guidelines]() and everyone is okay with the order. Please confirm with additional new members if their name and orchid is correct. 
+- Update `setup.py` file to have the new version number on the line `version=""`
+- Update `README.md` file to reflect the new changes of the release and version number 
+
+### Release Checklist
+
+Once the test is complete and at least three approvals are done make sure the following above is done.  
+The pyDARN lead developer or DAWG chairs should do this step; however, if you request to do it and get approval then go ahead! 
+
+1. **merge** the release branch in `master` and confirm the above updates are there
+2. [tag and release the code](https://github.com/SuperDARN/pydarn/releases/new)
+- tag: v<version number>
+- target: `master`
+- Release Title: pyDARN v<version number>
+- Description: Header "Version *number* - Release!" then add "pyDARN release v*number* includes the following features:" listing all the new changes to the code. Please see other [releases](https://github.com/SuperDARN/pydarn/releases) to keep with consistency
+- hit Publish release
+- Once a release has been made check on the [pyDARN Zenodo](https://zenodo.org/record/4558130) and look for the version you just released on the right hand side under **Versions**. Please note this may take some time. If this does not work Contact Lead Developer and DAWG Chairs on the matter 
```

Comment:
```suggestion
- Once a release has been made check on the [pyDARN Zenodo](https://zenodo.org/record/3727269) and look for the version you just released on the right hand side under **Versions**. Please note this may take some time. If this does not work Contact Lead Developer and DAWG Chairs on the matter 
```

Reply:
Did the link not work? I copied the general link

Reply:
The link worked but it was the link to v2.0.1, the one ending in 3727269 reverts to the latest version, it auto fills the URL to give 4558130 (v2.0.1) because it's the latest version.

Reply:
aaaa good catch